src/ParticipationStorage/ParticipationStorageInterface.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Phpab\Phpab\Storage;
+
+/**
+ * Stores the participation state of the user
+ */
+interface ParticipationStorageInterface
```

Comment:
Yeah weird, now they are both visible.

I'm nitpicking but I'd prefer:
- Analytics/AnalyticsInterface.php
  - Analytics/GoogleAnalytics.php
- Storage/StorageInterface.php
  - Storage/Cookie.php
- Participation/FilterInterface
  - ...

Choose whatever you want, like I said, it's nitpicking. Let's work towards that 1.0.0 release :dancers: